src/broker/cas_function.c

```diff
@@ -675,6 +689,8 @@ fn_execute_internal (SOCKET sock_fd, int argc, void **argv, T_NET_BUF * net_buf,
 		 (ret_code < 0) ? "error:" : "", err_number_execute, get_tuple_count (srv_handle), elapsed_sec,
 		 elapsed_msec, (client_cache_reusable == TRUE) ? " (CC)" : "",
 		 (srv_handle->use_query_cache == true) ? " (QC)" : "", eid_string);
+  ddl_node_type_find_and_set (srv_handle);
```

Comment:
It is difficult to imagine what the function **ddl_node_type_find_and_set** will do with just the function name. What about change the function name to be more intuitive? (`cub_ddl_log_mark_all_stmt_types_for_ddl` ?)

Reply:
ddl_node_type_find_and_set has been changed to cub_ddl_log_is_exist_ddl_stmt_type.
it returns ddl type if ddl type exists, and -1 if ddl type does not exist.